positron/comms/ui-frontend-openrpc.json

```diff
@@ -214,7 +214,8 @@
 						"type": "string"
 					}
 				}
-			]
+			],
+			"result": {}
```

Comment:
If we want to pass what the commands really return here so that command results can be plumbed back to R/Python, we _would_ need changes to the generator so let's punt that until we need it. For now, I have the rstudioapi functions (mostly) returning what they return in RStudio by setting that up in the R functions in ark.

Reply:
Yeah, it would be nice to have this but no motivating reason to do it now.